### ISSUE TITLE
7.1.17. Screen Name/Title: Creating an account - No Error Messages or Validation Feedback Present

### ISSUE THREAD

**Issue by jzbahrai** [cds-snc/notification-planning#2412]
Component: Creating an account form
Observation:
No Error Messages or Validation Feedback Present
The form contains required fields and uses required="", but there is no visible or programmatic feedback shown for validation errors.

(Attachment: 7.1.19. Validation Summary Alert Lacks Proper ARIA Role for SR.mp4)

Recommendation: 
Ensure error messages are announced to screen readers and associated with inputs via aria-describedby or inline messages. Also, indicate clearly on screen what the error is and how to correct it.

**Comment by whabanks** [comment]
Similar to #2474, if there is an issue here it's likely from `autofocus.js` and our old ways of managing focus. I re-tested this on this PR: https://github.com/cds-snc/notification-admin/pull/2410 and the banner is focused when the form is submitted with errors. The fields themselves properly announce the error hints as expected.

**Comment by smcmurtry** [comment]
Autofocus has been fixed and the `aria-describedby` that was missing from the TOU button was added here: https://github.com/cds-snc/notification-admin/pull/2404/files#diff-a7f550149122d467a344d9777dc9c59cdc623b3a69ab28e53b3d5853ce662047R52
